tests/exec-init/check-output/.gitignore

```diff
@@ -1,4 +1,4 @@
 target/
 Cargo.lock
 *.c
-build/
+build/
```

Comment:
Is the removal of the new line intentional?

Reply:
No (restored).